comp/host-profiler/collector/impl/converternoagent/converter_test.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build linux
+
+package converternoagent
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/confmap"
+)
+
+func TestConverterNoAgentConvert(t *testing.T) {
+	yaml := `
```

Comment:
Could we embed the yaml from `cmd/host-profiler/dist/host-profiler-config.yaml` for use in this test? That way if the default name is changed in `host-profiler-config.yaml`, we can ensure this functionality doesn't break